pdfminer/pdffont.py

```diff
@@ -648,18 +649,8 @@ def __init__(self, rsrcmgr, spec, strict=settings.STRICT):
         self.cidsysteminfo = dict_value(spec.get('CIDSystemInfo', {}))
         self.cidcoding = '%s-%s' % (resolve1(self.cidsysteminfo.get('Registry', b'unknown')).decode("latin1"),
                                     resolve1(self.cidsysteminfo.get('Ordering', b'unknown')).decode("latin1"))
-        try:
-            name = literal_name(spec['Encoding'])
-        except KeyError:
-            if strict:
-                raise PDFFontError('Encoding is unspecified')
-            name = 'unknown'
-        try:
-            self.cmap = CMapDB.get_cmap(name)
-        except CMapDB.CMapNotFound as e:
-            if strict:
-                raise PDFFontError(e)
-            self.cmap = CMap()
+        self.cmap = (spec, strict)
```

Comment:
This is the first time a `@property` is used in this repository. Do you have any special reason to introduce it here? Otherwise I would suggest keeping it simple without properties.

```suggestion
        self.cmap = self.get_cmap_from_spec(spec, strict)
```